backend/src/test/java/io/debezium/configserver/NativeMultipleKafkaConnectClustersIT.java

```diff
@@ -5,9 +5,9 @@
  */
 package io.debezium.configserver;
 
-import io.quarkus.test.junit.NativeImageTest;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
```

Comment:
Why not `Native` anymore?!

Reply:
Because Quarkus 2.8 deprecated `@NativeImageTest` in favor of `@QuarkusIntegrationTest`.